recipes/hmmlearn/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hmmlearn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 694646f8302bc6402925a4b6892f3a5ccede06d25f22157c18cfbdecdb748361
+
+build:
+  number: 0
+  skip: True  # [py37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - matplotlib >=1.1.1
+    - scikit-learn >=0.16,<0.20.0
+    - numpy >=1.9.3
+    - scipy >=0.16.0
+  run:
+    - python
+    - pytest >=2.6.0
+    - matplotlib >=1.1.1
```

Comment:
You can remove `matplotlib` as well as it's only needed for the examples which are not installed.

Reply:
Thanks @isuruf , I updated the recipe as suggested.